plugins/web3/starknet/src/lib.rs

```diff
@@ -0,0 +1,115 @@
+use beerus_core::{
+    config::Config,
+    lightclient::{
+        beerus::BeerusLightClient, ethereum::helios_lightclient::HeliosLightClient,
+        starknet::StarkNetLightClientImpl,
+    },
+};
+use bevy::{
+    app::{App, Plugin},
+    ecs::system::{Commands, Res},
+};
+use dotenv::dotenv;
+use prettytable::{format::consts::FORMAT_BOX_CHARS, table};
+use tokio::{runtime::Runtime, sync::mpsc::Sender};
+
+pub struct StarknetLightClientPlugin;
+
+impl Plugin for StarknetLightClientPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_event::<RpcRequest>();
+
+        if dotenv().is_ok() {
+            let config = Config::new_from_env().unwrap();
+
+            Self::print_config(&config);
+
+            let runtime = Runtime::new().unwrap();
+
+            let client = runtime.block_on(async move {
+                let ethereum_lightclient = HeliosLightClient::new(config.clone()).await.unwrap();
+                let starknet_lightclient = StarkNetLightClientImpl::new(&config.clone()).unwrap();
+
+                let mut client = BeerusLightClient::new(
+                    config.clone(),
+                    Box::new(ethereum_lightclient),
+                    Box::new(starknet_lightclient),
+                );
+
+                println!("Light Client initialized");
+
+                client
+            });
+
+            let runner = StarketLightClientRunner::new();
+            let tx = runner.start(client);
```

Comment:
TODO: This takes some time. Should be triggered from startup_system so that it doesn't block UI window to spawn.